### PR TITLE
Pr144 - Path correction

### PR DESCRIPTION
**Pull request by sbarguil**

### CODE REVIEW

File: yang/IETF-04/ietf-l3vpn-ntw.yang [hunk 1]
```diff
@@ -1893,7 +1939,7 @@ module ietf-l3vpn-ntw {
                     }
                     leaf-list tree-flavor {
                       type identityref {
-                        base vpn-common:multicast-tree-type;
```
Comment (boucadair) [inline yang/IETF-04/ietf-l3vpn-ntw.yang:1864]: Multicast tree type should be kept in the common module.

File: yang/IETF-04/ietf-l3vpn-ntw.yang [hunk 2]
```diff
@@ -123,6 +123,52 @@ module ietf-l3vpn-ntw {
...
+    "Base identity for static type.";
+   }
+
+   identity bsr-rp {
+    base multicast-rp-discovery-type;
+    description
+    "Base identity for BSR discovery type.";
+   }
+
+  identity multicast-tree-type {
```
Comment (boucadair) [inline yang/IETF-04/ietf-l3vpn-ntw.yang:149]: Should be moved to the common module. This is used also in the L2

File: yang/IETF-04/ietf-l3vpn-ntw.yang [hunk 3]
```diff
@@ -123,6 +123,52 @@ module ietf-l3vpn-ntw {
...
+    description
+    "Base identity for multicast tree type.";
+   }
+
+   identity ssm-tree-type {
```
Comment (boucadair) [inline yang/IETF-04/ietf-l3vpn-ntw.yang:154]: Should be kept in the common module.

File: yang/IETF-04/ietf-l3vpn-ntw.yang [hunk 4]
```diff
@@ -123,6 +123,52 @@ module ietf-l3vpn-ntw {
...
+    base multicast-tree-type;
```
Comment (boucadair) [inline yang/IETF-04/ietf-l3vpn-ntw.yang:155]:
```suggestion
    base ssm-tree-type;
```
this should be kept in the common module.

File: yang/IETF-04/ietf-l3vpn-ntw.yang [hunk 5]
```diff
@@ -123,6 +123,52 @@ module ietf-l3vpn-ntw {
...
+    description
+    "Identity for SSM tree type.";
+   }
+
+   identity asm-tree-type {
```
Comment (boucadair) [inline yang/IETF-04/ietf-l3vpn-ntw.yang:160]: To be maintained in the common module

File: yang/IETF-04/ietf-l3vpn-ntw.yang [hunk 6]
```diff
@@ -123,6 +123,52 @@ module ietf-l3vpn-ntw {
...
+    base multicast-tree-type;
```
Comment (boucadair) [inline yang/IETF-04/ietf-l3vpn-ntw.yang:161]: Redundant

File: yang/IETF-04/ietf-l3vpn-ntw.yang [hunk 7]
```diff
@@ -123,6 +123,52 @@ module ietf-l3vpn-ntw {
...
+    description
+    "Identity for ASM tree type.";
+   }
+
+   identity bidir-tree-type {
```
Comment (boucadair) [inline yang/IETF-04/ietf-l3vpn-ntw.yang:166]: to move to the common module

File: yang/IETF-04/ietf-l3vpn-ntw.yang (hunk 1 above)
Comment (sbarguil) [reply yang/IETF-04/ietf-l3vpn-ntw.yang:1864]: solved 951497867189b7da175218986a06ab8dc66ac80f

**Pull request merged**